config/routes.rb

```diff
@@ -1,6 +1,5 @@
 Rails.application.routes.draw do
   root 'users#index'
-
   resources :users, only: [:index, :show] do
     resources :posts, only: [:index, :show]
   end
```

Comment:
- [ ] Commendably, you have done a good job but there should be a route to show your work. So you need to set three different routes.
 first, for the post which could take this look
```
GET   /posts/new                   posts#new
POST   /posts(.:format)                     posts#create
```

Second one for the comments
```
   POST   /posts/:post_id/comments(.:format)     comments#create
 GET   /posts/:post_id/comments/new(.:format)     comments#new
```

and the last one for the likes
```
   POST   /posts/:post_id/likes(.:format)     likes#create
```

Once there are in place, it's a go for use.

Good job once again.